CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -99,30 +159,32 @@ def getOutputCell(cell, wsOut):
 
     Output:
     Cell objects from wsOut corresponding to coordinates of cell 
-    '''
+    """
 
     return wsOut[cell.coordinate]
 
 
-def getNonLinguisticCharacterCount(val, additionalCharactersToCatch = None, characterList = None):
-    '''
-    Check a string for how many of each kind of non-linguistic character it contains and return a dictionary mapping character to count.
+def getNonLinguisticCharacterCount(val, additionalCharactersToCatch=None, characterList=None):
```

Comment:
nit: It might be a bit more logical to reorder these arguments. `characterList` is the base and `additionalCharactersToCatch` is added on to that.